target/linux/bcm5862x/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Please add an SPDX license identifier on each new file you create (except *.patch).